packages/juce-blueprint/webpack.config.js

```diff
@@ -0,0 +1,59 @@
+const path = require('path');
+
+module.exports = env => {
+    return {
+        entry: './src/index.tsx',
+        output: {
+          path: __dirname + '/dist',
+          filename: 'index.js',
+          libraryTarget: "umd",
+          globalObject: "this",
+          sourceMapFilename: "[file].map",
+          devtoolModuleFilenameTemplate: 'webpack:///[absolute-resource-path]'
+        },
+        externals: {
+          react: {
+            root: 'React',
+            commonjs2: 'react',
+            commonjs: 'react',
+            amd: 'react'
+          },
+        },
+        devtool: "source-map",
+        resolve: {
+          extensions: [".ts", ".tsx", ".js"]
+        },
+        module: {
+          rules: [
+            {
+              test: /\.tsx?$/,
+              exclude: [
+                path.resolve(__dirname, 'dist'),
+                path.resolve(__dirname, 'node_modules')
+              ],
+              use: [
+                {
+                  loader: "awesome-typescript-loader",
```

Comment:
Using ATL as it seems to be the best option from my experiments. It generates all declarations, compiles/type-checks and then transpiles through babel and it's reasonably quick.